Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -305,4 +300,35 @@ public function getContentDimensionSource(): ContentDimensionSourceInterface
     {
         return $this->contentDimensionSource;
     }
+
+    private function enrich(EventsToPublish $eventsToPublish): EventsToPublish
```

Comment:
i would say `enrichEventsToPublishWithMetadata` and the comment could be just a doc comment then